cli.js

```diff
@@ -53,10 +63,18 @@ Usage:\n\
         allowMultiple: true,
       };
     });
+
+    // Attempt to load custom extractor from path.
+    let customExtractor;
+    if (addExtractor) {
+      customExtractor = require(addExtractor);
```

Comment:
Let's do a try/catch here so we can provide some error message if the path doesn't exist or there some problem w/the import.

Reply:
Hey @adampash, in the cli.js file, the overall functionality is already wrapped in a try/catch which spits out the error if the file cannot be imported.